zebra/zebra_vxlan_if.c

```diff
@@ -521,17 +521,38 @@ static int zebra_vxlan_if_add_update_vni(struct zebra_if *zif,
 
 	if (old_vni->access_vlan != vni->access_vlan) {
 		if (IS_ZEBRA_DEBUG_VXLAN)
-			zlog_debug(
-				"vxlan %s updating vni(%d, %d) -> vni(%d, %d)",
-				zif->ifp->name, old_vni->vni,
-				old_vni->access_vlan, vni->vni,
-				vni->access_vlan);
+			zlog_debug("%s vxlan %s updating vni(%d, %d) -> vni(%d, %d)",
+				   __func__, zif->ifp->name, old_vni->vni,
+				   old_vni->access_vlan, vni->vni,
+				   vni->access_vlan);
 
 		zebra_evpn_vl_vxl_deref(old_vni->access_vlan, old_vni->vni,
 					zif);
 		zebra_evpn_vl_vxl_ref(vni->access_vlan, vni->vni, zif);
 		zebra_vxlan_if_update_vni(zif->ifp, vni, ctx);
 		zebra_vxlan_vni_free(old_vni);
+	} else {
+		int ret;
+
+		ret = zebra_evpn_vl_vxl_bridge_lookup(vni->access_vlan, zif);
+		/* Here ret value 0 implied bridge vlan mapping is not present
+		 * repopulated. Ignore ret value 1 as it means vlan mapping is
+		 * present in bridge table.
+		 */
+		if (ret < 0) {
+			zlog_debug("%s vxlan %s vni %u has error accessing bridge table.",
```

Comment:
unguarded zlog_debug(), I think we should fix it?

Reply:
my bad will fix it.